src/Common/src/Interop/Windows/kernel32/Interop.GetFullPathNameW.cs

```diff
@@ -13,6 +13,6 @@ internal partial class Kernel32
         /// WARNING: This method does not implicitly handle long paths. Use GetFullPathName or PathHelper.
         /// </summary>
         [DllImport(Libraries.Kernel32, SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false, ExactSpelling = true)]
-        internal static extern unsafe uint GetFullPathNameW(char* path, uint numBufferChars, SafeHandle buffer, IntPtr mustBeZero);
+        internal static unsafe extern uint GetFullPathNameW(string path, uint bufferLength, char* buffer, IntPtr mustBeZero);
```

Comment:
Nit, but ideally this would be private and we'd expose (in this same file) a wrapper function that dropped the useless last parameter.